1- Faça um Programa que peça dois números e imprima o maior deles.

In [1]:
x = int(input("Digite um valor para x: "))
y = int(input("Digite outro valor para y: "))
if x > y:
    print(f"O maior valor é {x}")
elif y > x:
    print(f"O maior valor é {y}")
else:
    print("Os números são iguais")


Digite um valor para x: 5
Digite outro valor para y: 5
Os números são iguais


2 - Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [11]:
x = input ('Digite uma letra: ')

if x == 'a' or x == 'e' or x == 'i' or x == 'o' or x == 'u':
    print ('A letra é uma vogal')
else:
    print ('A letra é uma consoante')

Digite uma letra: a
A letra é uma vogal


3 - Faça um programa para a leitura de duas notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e apresentar:
- A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
- A mensagem "Reprovado", se a média for menor do que sete;
- A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [24]:
n1 = input('Digite o valor da primeira nota: ')
n2 = input('Digite o valor da segunda nota: ')

media = (int(n1) + int(n2)) / 2

if media >= 7 and media < 10:     
    print (f'Média {media} - Aprovado') 
elif media == 10:
    print (f'Média {media} - Aprovado com Distinção')
else:
    print (f'Média {media} - Reprovado')

Digite o valor da primeira nota: 5
Digite o valor da segunda nota: 4
Média 4.5 - Reprovado


4 - Faça um Programa que leia três números e mostre-os em ordem decrescente.

In [32]:
lista = []
i = 0
while i < 3:
    x = input('Digite um valor: ')
    lista.append(x)
    i += 1
lista.sort(reverse=True)
print(lista)

Digite um valor: 4
Digite um valor: 5
Digite um valor: 6
['6', '5', '4']


5 - As Organizações Tabajara resolveram dar um aumento de salário aos seus
colaboradores e lhe contrataram para desenvolver o programa que calculará os
reajustes.
Faça um programa que recebe o salário de um colaborador e o reajuste segundo o
seguinte critério, baseado no salário atual:
- salários até R$ 280,00 (incluindo) : aumento de 20%
- salários entre R$ 280,00 e R$ 700,00 : aumento de 15%
- salários entre R$ 700,00 e R$ 1500,00 : aumento de 10%
- salários de R$ 1500,00 em diante : aumento de 5% Após o aumento ser realizado,
informe na tela:
- o salário antes do reajuste;
- o percentual de aumento aplicado;
- o valor do aumento;
- o novo salário, após o aumento.

In [38]:
salario = int(input('Digite o valor do salario: R$ '))

if salario <= 280:
      por = (20 / 100.0) * salario
      resultado = salario + por
      print ('Percentual aumentado: %',20)
      print ('O valor do aumento é: R$',por)
      print ('Com o reajuste, o seu salário é: R$',resultado)

#Se o salario é maior que 280 e menor que 700

elif salario > 280.00 and salario <= 700.00:
        por = (15 / 100.0) * salario
        resultado = salario + por
        print ('Percentual aumentado: %',15)
        print ('O valor do aumento é: R$',por)
        print ('Com o reajuste, o seu salário é: R$',resultado)

#Se o salario é maior que 700 e menor que 1500

elif salario > 700.00 and salario <= 1500.00:
        por = (10 / 100.0) * salario
        resultado = salario + por
        print ('Percentual aumentado: %',10)
        print ('O valor do aumento é: R$',por)
        print ('Com o reajuste, o seu salário é: R$',resultado)

#Se o salario é maior que 1500

elif salario > 1500.00:
        por = (5 / 100.0) * salario
        resultado = salario + por
        print ('Percentual aumentado: %',5)
        print ('O valor do aumento é: R$',por)
        print ('Com o reajuste, o seu salário é: R$',resultado)

Digite o valor do salario: R$ 1200
Percentual aumentado: % 10
O valor do aumento é: R$ 120.0
Com o reajuste, o seu salário é: R$ 1320.0


6 - Faça um Programa que leia um número e exiba o dia correspondente da semana.
(1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido.

In [40]:
dia = int(input('Digite o número do dia desejado: '))
if dia == 1:
    print('Você digitou o dia referente à Domingo')
elif dia == 2:
    print('Você digitou o dia referente à Segunda-Feira')
elif dia == 3:
    print('Você digitou o dia referente à Terça-Feira')
elif dia == 4:
    print('Você digitou o dia referente à Quarta-Feira')
elif dia == 5:
    print('Você digitou o dia referente à Quinta-Feira')
elif dia == 6:
    print('Você digitou o dia referente à Sexta-Feira')
elif dia == 7:
    print('Você digitou o dia referente à Sábado')
else:
    print('Você digitou um número inválido')

Digite o número do dia desejado: 9
Você digitou um número inválido


7 - Faça um programa que lê as duas notas parciais obtidas por um aluno numa
disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos
obedece à tabela abaixo:
 Média de Aproveitamento Conceito
 Entre 9.0 e 10.0 A
 Entre 7.5 e 9.0 B
 Entre 6.0 e 7.5 C
 Entre 4.0 e 6.0 D
 Entre 4.0 e zero E
O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a
mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito
for D ou E.

In [50]:
n1 = int(input('Digite o valor da primeira nota: '))
n2 = int(input('Digite o valor da primeira nota: '))
media = ((n1 + n2)/2)

#Seta o Conceito 
if media >= 9:
    conceito = "A"
elif media >= 7.5 and media < 9:
    conceito = "B"
elif media >= 6 and media < 7.5:
    conceito = "C"
elif media >= 4 and media <6:
    conceito = "D"
else:
    conceito = "E"
    
#Seta o Status    
if conceito == "A" or "B" or "C":
    status = "Aprovado"
else:
    status = "Reprovado"
    
print(f"Média do aluno: {media}")
print(f"Conceito do aluno: {conceito}")
print(f"Status do aluno: {status}")

Digite o valor da primeira nota: 5
Digite o valor da primeira nota: 4
Média do aluno: 4.5
Conceito do aluno: D
Status do aluno: Aprovado


8 - Faça um Programa que peça os 3 lados de um triângulo. O programa deverá
informar se os valores podem ser um triângulo. Indique, caso os lados formem um
triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:
- Três lados formam um triângulo quando a soma de quaisquer dois lados for maior
que o terceiro;
- Triângulo Equilátero: três lados iguais;
- Triângulo Isósceles: quaisquer dois lados iguais;
- Triângulo Escaleno: três lados diferentes;

In [ ]:
a = int(input('Digite o valor para o lado a: '))
b = int(input('Digite o valor para o lado b: '))
c = int(input('Digite o valor para o lado c: '))

#Verifica se é um triângulo
if a + b > c and a + c > b and b + c > a:
    #Verifica se é Equilátero
    if a == b or a == c or b == c:
        print('O Triângulo é Isósceles')
        #Verifica se é Isósceles
        if a == b and b == c:
            print('O Triângulo também é Equilátero')
    else:
        print('O Triângulo é Escaleno')
else:
    print('Os valores digitados não formam um triângulo')

9 - Faça um programa que calcule as raízes de uma equação do segundo grau, na
forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as
consistências, informando ao usuário nas seguintes situações:
- Se o usuário informar o valor de A igual a zero, a equação não é do segundo grau e
o programa não deve fazer pedir os demais valores, sendo encerrado;
- Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao
usuário e encerre o programa;
- Se o delta calculado for igual a zero a equação possui apenas uma raiz real;
informe-a ao usuário;
- Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;


In [9]:
import math
a = int(input('Digite o valor para a: '))
b = int(input('Digite o valor para b: '))
c = int(input('Digite o valor para c: '))
delta = 0
#Verifica o valor de a
if a == 0:
    print(' equação não é do segundo grau')
else:
    delta = math.pow(b,2) - (4 * a *c)
    if delta < 0:
        print('A esquação não possui raízes')
    else:
        if delta == 0:
            print('A equação possui uma única raíz')
            r = ((-1)*b + math.sqrt(delta))/(2*a)
            print(r)
        else:
            print ("delta = ",delta," a equacao possui duas raizes")
            r1 = ((-1)*b + math.sqrt(delta))/(2*a)
            r2 = ((-1)*b - math.sqrt(delta))/(2*a)
            print ("raiz1 da equacao = ", r1)
            print ("raiz2 da equacao = ", r2)
    
    

Digite o valor para a: 1
Digite o valor para b: 4
Digite o valor para c: 4
A equação possui uma única raíz
-2.0


10 - Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao
usuário a valor do saque e depois informar quantas notas de cada valor serão
fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é
de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a
quantidade de notas existentes na máquina.
- Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de
100, uma nota de 50, uma nota de 5 e uma nota de 1;
- Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de
100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.


In [1]:
valor = int(input('Digite o valor a ser sacado: '))
nota1 = 0
nota5 = 0
nota10 = 0
nota50 = 0
nota100 = 0

nota100 = valor // 100
valor -= nota100 * 100

nota50 = valor // 50
valor -= nota50 * 50

nota10 = valor // 10
valor -= nota10 * 10

nota5 = valor // 5
valor -= nota5 * 5

nota1 = valor // 1
valor -= nota1 * 1

print (valor)
    
    
        
print(f'Nota de 100: {nota100}')  
print(f'Nota de 50: {nota50}')
print(f'Nota de 10: {nota10}')
print(f'Nota de 5: {nota5}')
print(f'Nota de 1: {nota1}')
    
    


Digite o valor a ser sacado: 1111
0
Nota de 100: 11
Nota de 50: 0
Nota de 10: 1
Nota de 5: 0
Nota de 1: 1


11 - Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As
perguntas são:
"Telefonou para a vítima?"
"Esteve no local do crime?"
"Mora perto da vítima?"
"Devia para a vítima?"
"Já trabalhou com a vítima?"
O programa deve no final emitir uma classificação sobre a participação da pessoa no
crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada
como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário,
ele será classificado como "Inocente"

In [14]:
print('Responda SIM ou ou NAO')

x1 = input('Você telefonou para a vítima? ')
x2 = input('Esteve no local do crime? ')
x3 = input('Mora perto da vítima? ')
x4 = input('Devia para a vítima? ')
x5 = input('Já trabalhou com a vítima? ')
classificacao = 'Inocente'

#Verifica a classiicação
if x2 == 'SIM':
    classificacao = "Suspeita"
elif x3 == 'SIM' and x4 == 'SIM':
    classificacao = 'Cumplice'
elif x5 == 'SIM':
    classificacao = 'Assasino'

    
print(f'O entrevistado é: {classificacao}')

Responda SIM ou ou NAO
Você telefonou para a vítima? SIM
Esteve no local do crime? IM
Mora perto da vítima? SIM
Devia para a vítima? SIM
Já trabalhou com a vítima? SIM
O entrevistado é: Cumplice


12 - Uma fruteira está vendendo frutas com a seguinte tabela de preços:
 Até 5 Kg Acima de 5 Kg
Morango R$ 2,50 por Kg R$ 2,20 por Kg
Maçã R$ 1,80 por Kg R$ 1,50 por Kg
Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar
R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um
algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de
maças adquiridas e escreva o valor a ser pago pelo cliente

13 - Faça um programa que peça uma nota, entre zero e dez. Mostre uma mensagem
caso o valor seja inválido e continue pedindo até que o usuário informe um valor
válido.

In [10]:
condicional = True
while condicional:
    valor = int(input('Digite um valor: '))
    if (valor >= 0 and valor <= 10):
        print('Valor aceito!')
        condicional = False     
    else:
        print('Valor incorreto.')

Digite um valor: 11
Valor incorreto.
Digite um valor: -1
Valor incorreto.
Digite um valor: 1
Valor aceito!


14 - Faça um programa que leia e valide as seguintes informações:
Nome: maior que 3 caracteres;
Idade: entre 0 e 150;
Salário: maior que zero;
Sexo: 'f' ou 'm';
Estado Civil: 's', 'c', 'v', 'd';


In [15]:
#Nome
nome = input('Digite seu nome: ')
if len(nome) > 3 :
    print('Nome válido!')
else:
     print('Nome inválido.')
        
#Idade
idade = int(input('Digite a sua idade: '))
if idade > 0 and idade <= 150:
    print('Idade válida!')
else: 
    print('Idade inválida.')
    
#Salário
salario = float(input('Digite o seu salario: '))
if salario > 0:
    print('Salário válido!')
else:
    print('Salario inválido.')
    
#Sexo 
sexo = input('Digite o sexo -> f para feminino | m para masculino: ').lower()
if sexo == 'm' or sexo =='f':
    print('Sexo válido!')
else:
    print('Sexo inválido.')

#Estado Civil    
estado_civil = input('Digite o estado civil -> s para solteiro | c para casado: | v para viúvo | d para divorciado').lower()
if estado_civil == 's' or estado_civil == 'c' or estado_civil == 'v' or estado_civil == 'd':
    print('Estado Civil válido!')
else:
    print('Estado Civil inválido.')
    



Digite seu nome: Lucas
Nome válido!
Digite a sua idade: 23
Idade válida!
Digite o seu salario: 1300
Salário válido!
Digite o sexo -> f para feminino | m para masculino: m
Sexo válido!
Digite o estado civil -> s para solteiro | c para casado: | v para viúvo | d para divorciados
Estado Civil válido!


15 - Faça um programa que leia 5 números e informe o maior número.

In [21]:
lista = []

i= 0
while i < 5:
    x = int(input('Digite um número: '))
    lista.append(x)
    i += 1
    
print('O maior número digitado é: ', max(lista))


Digite um número: 60
Digite um número: 55
Digite um número: 44
Digite um número: 10
Digite um número: 11
O maior número digitado é:  60


16 - Faça um programa que leia 5 números e informe a soma e a média dos números.

In [24]:
lista = []
soma = 0
media = 0
i= 0
while i < 5:
    x = int(input('Digite um número: '))
    lista.append(x)
    i += 1
for numero in lista:
    soma += numero
media = (soma / 5)
print(f'Soma: {soma}')
print(f'Média: {media}')

Digite um número: 10
Digite um número: 20
Digite um número: 30
Digite um número: 40
Digite um número: 50
Soma: 150
Média: 30.0


17 - Faça um programa que imprima na tela apenas os números ímpares entre 1 e 50.

In [29]:
lista = []
i = 1 
while i <= 50:    
    if(i % 2 == 1):
        lista.append(i)
    i += 1
        
print(lista)

[1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47, 49]


18 - A série de Fibonacci é formada pela seqüência 1,1,2,3,5,8,13,21,34,55,... Faça um
programa capaz de gerar a série até o n−ésimo termo.

In [39]:
n = int(input("Digite o número de termos: "))
a, b = 0, 1
while b < n:
       print (b)
       a, b = b, a+b


Digite o número de termos: 100
1
1
2
3
5
8
13
21
34
55
89


19 - Faça um programa que calcule o fatorial de um número inteiro fornecido pelo
usuário. Ex.: 5!=5.4.3.2.1=120

In [10]:
n = int(input('Informe o número de termos desejados para realizar a fatorial: '))
i = 1
fatorial = 1
while i <= n:
    fatorial = fatorial * i
    i += 1
print(f'Fatorial de {n} = {fatorial}')

Informe o número de termos desejados para realizar a fatorial: 6
Fatorial de 6 = 720


20 - O Sr. Manoel Joaquim acaba de adquirir uma panificadora e pretende implantar
a metodologia da tabelinha, que já é um sucesso na sua loja de 1,99. Você foi
contratado para desenvolver o programa que monta a tabela de preços de pães, de
1 até 50 pães, a partir do preço do pão informado pelo usuário, conforme o exemplo
abaixo:
Preço do pão: R$ 0.18
Panificadora Pão de Ontem - Tabela de preços
1 - R$ 0.18
2 - R$ 0.36
...
50 - R$ 9.00

In [19]:
i = 1
while i <= 50:
    print(f'Qtd {i} = R$ %5.2f' % (i* 1.99))
    i += 1

Qtd 1 = R$  1.99
Qtd 2 = R$  3.98
Qtd 3 = R$  5.97
Qtd 4 = R$  7.96
Qtd 5 = R$  9.95
Qtd 6 = R$ 11.94
Qtd 7 = R$ 13.93
Qtd 8 = R$ 15.92
Qtd 9 = R$ 17.91
Qtd 10 = R$ 19.90
Qtd 11 = R$ 21.89
Qtd 12 = R$ 23.88
Qtd 13 = R$ 25.87
Qtd 14 = R$ 27.86
Qtd 15 = R$ 29.85
Qtd 16 = R$ 31.84
Qtd 17 = R$ 33.83
Qtd 18 = R$ 35.82
Qtd 19 = R$ 37.81
Qtd 20 = R$ 39.80
Qtd 21 = R$ 41.79
Qtd 22 = R$ 43.78
Qtd 23 = R$ 45.77
Qtd 24 = R$ 47.76
Qtd 25 = R$ 49.75
Qtd 26 = R$ 51.74
Qtd 27 = R$ 53.73
Qtd 28 = R$ 55.72
Qtd 29 = R$ 57.71
Qtd 30 = R$ 59.70
Qtd 31 = R$ 61.69
Qtd 32 = R$ 63.68
Qtd 33 = R$ 65.67
Qtd 34 = R$ 67.66
Qtd 35 = R$ 69.65
Qtd 36 = R$ 71.64
Qtd 37 = R$ 73.63
Qtd 38 = R$ 75.62
Qtd 39 = R$ 77.61
Qtd 40 = R$ 79.60
Qtd 41 = R$ 81.59
Qtd 42 = R$ 83.58
Qtd 43 = R$ 85.57
Qtd 44 = R$ 87.56
Qtd 45 = R$ 89.55
Qtd 46 = R$ 91.54
Qtd 47 = R$ 93.53
Qtd 48 = R$ 95.52
Qtd 49 = R$ 97.51
Qtd 50 = R$ 99.50


21 - O Sr. Manoel Joaquim possui uma grande loja de artigos de R$ 1,99, com cerca
de 10 caixas. Para agilizar o cálculo de quanto cada cliente deve pagar ele
desenvolveu um tabela que contém o número de itens que o cliente comprou e ao
lado o valor da conta. Desta forma a atendente do caixa precisa apenas contar
quantos itens o cliente está levando e olhar na tabela de preços. Você foi contratado
para desenvolver o programa que monta esta tabela de preços, que conterá os
preços de 1 até 50 produtos, conforme o exemplo abaixo:
Lojas Quase Dois - Tabela de preços
1 - R$ 1.99
2 - R$ 3.98
...
50 - R$ 99.50

In [20]:
valor = float(input('Informe o valor de pães desejado: '))
i = 1
while i <= 50:
    print(f'Qtd {i} = R$ %5.2f' % (i* valor))
    i += 1

Informe o valor de pães desejado: 2
Qtd 1 = R$  2.00
Qtd 2 = R$  4.00
Qtd 3 = R$  6.00
Qtd 4 = R$  8.00
Qtd 5 = R$ 10.00
Qtd 6 = R$ 12.00
Qtd 7 = R$ 14.00
Qtd 8 = R$ 16.00
Qtd 9 = R$ 18.00
Qtd 10 = R$ 20.00
Qtd 11 = R$ 22.00
Qtd 12 = R$ 24.00
Qtd 13 = R$ 26.00
Qtd 14 = R$ 28.00
Qtd 15 = R$ 30.00
Qtd 16 = R$ 32.00
Qtd 17 = R$ 34.00
Qtd 18 = R$ 36.00
Qtd 19 = R$ 38.00
Qtd 20 = R$ 40.00
Qtd 21 = R$ 42.00
Qtd 22 = R$ 44.00
Qtd 23 = R$ 46.00
Qtd 24 = R$ 48.00
Qtd 25 = R$ 50.00
Qtd 26 = R$ 52.00
Qtd 27 = R$ 54.00
Qtd 28 = R$ 56.00
Qtd 29 = R$ 58.00
Qtd 30 = R$ 60.00
Qtd 31 = R$ 62.00
Qtd 32 = R$ 64.00
Qtd 33 = R$ 66.00
Qtd 34 = R$ 68.00
Qtd 35 = R$ 70.00
Qtd 36 = R$ 72.00
Qtd 37 = R$ 74.00
Qtd 38 = R$ 76.00
Qtd 39 = R$ 78.00
Qtd 40 = R$ 80.00
Qtd 41 = R$ 82.00
Qtd 42 = R$ 84.00
Qtd 43 = R$ 86.00
Qtd 44 = R$ 88.00
Qtd 45 = R$ 90.00
Qtd 46 = R$ 92.00
Qtd 47 = R$ 94.00
Qtd 48 = R$ 96.00
Qtd 49 = R$ 98.00
Qtd 50 = R$ 100.00


22 - O Sr. Manoel Joaquim expandiu seus negócios para além dos negócios de 1,99 e
agora possui uma loja de conveniências. Faça um programa que implemente uma
caixa registradora rudimentar. O programa deverá receber um número
desconhecido de valores referentes aos preços das mercadorias. Um valor zero deve
ser informado pelo operador para indicar o final da compra. O programa deve então
mostrar o total da compra e perguntar o valor em dinheiro que o cliente forneceu,
para então calcular e mostrar o valor do troco. Após esta operação, o programa
deverá voltar ao ponto inicial, para registrar a próxima compra. A saída deve ser
conforme o exemplo abaixo:
Lojas Tabajara
Produto 1: R$ 2.20
Produto 2: R$ 5.80
Produto 3: R$ 0
Total: R$ 9.00
Dinheiro: R$ 20.00
Troco: R$ 11.00
...

In [ ]:
produtos = []
total = 0
print('....::::Iniciando Nova venda:::::....')
print('Digite 0 para finalizar a venda\n')
while True:    
    valor = float(input('Informe o valor do produto: '))
    if valor != 0:
        produtos.append(valor)
    else:
        print('\n\nCalculando Venda ...')
        if len(produtos) == 0:
            print('Lista de Vazia!')
        else:
            i = 1
            for produto in produtos:
                print(f'Produto {i} = R$ %5.2f' % (produto))
                total += produto                        
                i += 1  
        print(f'\nTotal à pagar: R$ {total}')  
        pago = float(input('Informe o valor pago ao caixa: '))
        troco = pago - total
        print(f'\n\nTroco: R$ {troco}')
        print('\nCaixa finalizado!\n\n....::::Iniciando Nova venda:::::....')
        produtos.clear()


....::::Iniciando Nova venda:::::....
Digite 0 para finalizar a venda

Informe o valor do produto: 5
Informe o valor do produto: 3
Informe o valor do produto: 7
Informe o valor do produto: 9
Informe o valor do produto: 3
Informe o valor do produto: 0


Calculando Venda ...
Produto 1 = R$  5.00
Produto 2 = R$  3.00
Produto 3 = R$  7.00
Produto 4 = R$  9.00
Produto 5 = R$  3.00

Total à pagar: R$ 27.0
Informe o valor pago ao caixa: 30


Troco: R$ 3.0

Caixa finalizado!

....::::Iniciando Nova venda:::::....


23 -  Foi feita uma estatística em cinco cidades brasileiras para coletar dados sobre
acidentes de trânsito. Foram obtidos os seguintes dados:
- Código da cidade;
- Número de veículos de passeio (em 1999);
- Número de acidentes de trânsito com vítimas (em 1999). Deseja-se saber:
- Qual o maior e menor índice de acidentes de transito e a que cidade pertence;
- Qual a média de veículos nas cinco cidades juntas;
- Qual a média de acidentes de trânsito nas cidades com menos de 2.000 veículos de
passeio.

24 -  Faça um programa que receba o valor de uma dívida e mostre uma tabela com
os seguintes dados: valor da dívida, valor dos juros, quantidade de parcelas e valor
da parcela.
Os juros e a quantidade de parcelas seguem a tabela abaixo:
Quantidade de Parcelas % de Juros sobre o valor inicial da dívida

In [6]:
divida = float(input('Digite o valor da dívida: '))


print(f'Valor da Dívida: {divida*1}', f'\tValor dos Juros: R$ {0000.0}', '\tQtd. Parcelas: 0', f'\tValor da Parcela: R$ {divida}')
print(f'Valor da Dívida: {divida*1.1}', f'\tValor dos Juros: R$ {divida*0.1}', '\tQtd. Parcelas: 3', f'\tValor da Parcela: R$ {(divida*1.1)/3}')
print(f'Valor da Dívida: {divida*1.15}', f'\tValor dos Juros: R$ {divida*0.15}', '\tQtd. Parcelas: 6', f'\tValor da Parcela: R$ {(divida*1.15)/6}')
print(f'Valor da Dívida: {divida*1.20}', f'\tValor dos Juros: R$ {divida*0.20}', '\tQtd. Parcelas: 9', f'\tValor da Parcela: R$ {(divida*1.20)/9}')
print(f'Valor da Dívida: {divida*1.25}', f'\tValor dos Juros: R$ {divida*0.25}', '\tQtd. Parcelas: 12', f'\tValor da Parcela: R$ {(divida*1.25)/12}')

Digite o valor da dívida: 1200
Valor da Dívida: 1200.0 	Valor dos Juros: R$ 0.0 	Qtd. Parcelas: 0 	Valor da Parcela: R$ 1200.0
Valor da Dívida: 1320.0 	Valor dos Juros: R$ 120.0 	Qtd. Parcelas: 3 	Valor da Parcela: R$ 440.0
Valor da Dívida: 1380.0 	Valor dos Juros: R$ 180.0 	Qtd. Parcelas: 6 	Valor da Parcela: R$ 230.0
Valor da Dívida: 1440.0 	Valor dos Juros: R$ 240.0 	Qtd. Parcelas: 9 	Valor da Parcela: R$ 160.0
Valor da Dívida: 1500.0 	Valor dos Juros: R$ 300.0 	Qtd. Parcelas: 12 	Valor da Parcela: R$ 125.0


25 - Faça um programa que mostre os n termos da Série a seguir:
 S = 1/1 + 2/3 + 3/5 + 4/7 + 5/9 + ... + n/m.
Imprima no final a soma da série.

In [14]:
n = int(input('Digite o termo desejado: '))
a, b = 1, 1
soma = 0
i = 0
while i < n:
      print(f'S = {a}/{b}')
      soma += (a/b)
      a += 1
      b += 2
      i += 1
print(f'Soma = {soma}')      

Digite o termo desejado: 100
S = 1/1
S = 2/3
S = 3/5
S = 4/7
S = 5/9
S = 6/11
S = 7/13
S = 8/15
S = 9/17
S = 10/19
S = 11/21
S = 12/23
S = 13/25
S = 14/27
S = 15/29
S = 16/31
S = 17/33
S = 18/35
S = 19/37
S = 20/39
S = 21/41
S = 22/43
S = 23/45
S = 24/47
S = 25/49
S = 26/51
S = 27/53
S = 28/55
S = 29/57
S = 30/59
S = 31/61
S = 32/63
S = 33/65
S = 34/67
S = 35/69
S = 36/71
S = 37/73
S = 38/75
S = 39/77
S = 40/79
S = 41/81
S = 42/83
S = 43/85
S = 44/87
S = 45/89
S = 46/91
S = 47/93
S = 48/95
S = 49/97
S = 50/99
S = 51/101
S = 52/103
S = 53/105
S = 54/107
S = 55/109
S = 56/111
S = 57/113
S = 58/115
S = 59/117
S = 60/119
S = 61/121
S = 62/123
S = 63/125
S = 64/127
S = 65/129
S = 66/131
S = 67/133
S = 68/135
S = 69/137
S = 70/139
S = 71/141
S = 72/143
S = 73/145
S = 74/147
S = 75/149
S = 76/151
S = 77/153
S = 78/155
S = 79/157
S = 80/159
S = 81/161
S = 82/163
S = 83/165
S = 84/167
S = 85/169
S = 86/171
S = 87/173
S = 88/175
S = 89/177
S = 90/179
S = 91/181
S = 92/183
S = 93/185
S = 94/187
S